R/utils-json.R

```diff
@@ -218,6 +219,12 @@ getApprovals <- function(data, chain_nm, legend_nm, appr_var_all, month=NULL, po
   return(approvals_all)
 }
 
+subsetByMonth <- function(pts, onlyMonth) {
```

Comment:
doesn't this function already exist - https://github.com/USGS-R/repgen/blob/master/R/uvhydrograph-data.R#L165-L170? Looks like it needs to be pulled out into `utils-shared.R`